src/hadoop-data-node/config/hadoop-data-node.md

```diff
@@ -0,0 +1,44 @@
+## Hadoop data node section parser
+
+- [Default Configuration](#D_Config)
```

Comment:
lack 
1 first deployed user how to do: step by step. [step's could refer some deployment doc]
2 upgrading user how to do: step by step. [step's could refer some deployment doc]

Reply:
@YanjieGao 
I think in our design, this doc is only for per service configuration introduction. Not for maintance or other purpose.
i.e.
https://github.com/Microsoft/pai/blob/master/src/rest-server/config/rest-server.md
https://github.com/Microsoft/pai/blob/master/src/grafana/config/grafana.md

Of course, I will update hdfs docs after this PR

Reply:
ok